### PR TITLE
fix: mocks should match reality

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

In the recent Dashboard Template fangling the test mocks had got out of step with reality, so while the tests passed, the system wasn't working as expected (behind a flag)

## Changes

Ensure the OG template is always available

## How did you test this code?

developer tests and 👀

### CODE REVIEW

File: posthog/api/dashboards/dashboard_templates.py [hunk 1]
```diff
@@ -84,7 +84,16 @@ def repository(self, request: request.Request, **kwargs) -> response.Response:
     def _load_repository_listing() -> List[Dict]:
         url = "https://raw.githubusercontent.com/PostHog/templates-repository/main/dashboards/dashboards.json"
         loaded_templates: List[Dict] = json.loads(requests.get(url).text)
-        return loaded_templates
+        return [
+            # The OG template is hard-coded, not served by the repository, so added in to the template listing here
+            {
+                "name": "Product analytics",
+                "url": "some url",
```
Comment (Twixes) [inline posthog/api/dashboards/dashboard_templates.py:91]: Feels wrong to have "some url" as a string in production code…

File: posthog/api/dashboards/dashboard_templates.py [hunk 2]
```diff
@@ -84,7 +84,16 @@ def repository(self, request: request.Request, **kwargs) -> response.Response:
     def _load_repository_listing() -> List[Dict]:
         url = "https://raw.githubusercontent.com/PostHog/templates-repository/main/dashboards/dashboards.json"
         loaded_templates: List[Dict] = json.loads(requests.get(url).text)
-        return loaded_templates
+        return [
+            # The OG template is hard-coded, not served by the repository, so added in to the template listing here
```
Comment (Twixes) [inline posthog/api/dashboards/dashboard_templates.py:88]: Why keep this hard-coded instead of adding to the repository?

Comment (pauldambra) [reply posthog/api/dashboards/dashboard_templates.py:88]: We use it in tests and in initial team setup. So ran into a chicken and egg problem of having to load templates to do anything.... but that's in my brain not the code

File: posthog/api/dashboards/dashboard_templates.py (hunk 1 above)
Comment (pauldambra) [reply posthog/api/dashboards/dashboard_templates.py:91]: 🙈

**Review by Twixes — APPROVED** [review]
LGTM

**Pull request merged**